examples/kotlin-multiplatform/build.gradle

```diff
@@ -30,6 +30,12 @@ kotlin {
 
     applyDefaultHierarchyTemplate()
 
+    targets.configureEach {
+        compilations.configureEach {
+            kotlinOptions.allWarningsAsErrors = true
```

Comment:
Done.
But configuring `compilerOptions` in Groovy is not as handy as in Kotlin.
It is not possible to call `compilerOptions.configure {}` function because of resolution ambiguity between `fun configure(configuration: T.() -> Unit)` and `fun configure(configuration: Action<T>)`.
So the `compilerOptions.options` needs to be configured directly.